compiler/env/OMRClassEnv.hpp

```diff
@@ -141,6 +143,16 @@ class OMR_EXTENSIBLE ClassEnv
     */
    const TR::TypeLayout* enumerateFields(TR::Region& region, TR_OpaqueClassBlock * clazz, TR::Compilation *comp) { return NULL; }
 
+   /**
+    * @brief Determines if a class hass less than two concrete classes in the
+    * list of subclasses. A class is concrete if it is not an interface class.
+    * 
+    * @param subClasses List of subclasses for current class.
+    * 
+    * @return boolean variable that indicates if a class contains zero or one
```

Comment:
@return Returns 'true' if the given list of classes contains less than 2 concrete classs and false otherwise